minidump-common/src/format.rs

```diff
@@ -845,9 +845,9 @@ pub enum ArmRegisterNumbers {
 /// aarch64 floating point state
 #[derive(Clone, Pread, SizeWith)]
 pub struct FLOATING_SAVE_AREA_ARM64 {
```

Comment:
FYI, Breakpad defines two versions of this struct, an "old" and a "new":
https://chromium.googlesource.com/breakpad/breakpad/+/88d8114fda3e4a7292654bd6ac0c34d6c88a8121/src/google_breakpad/common/minidump_cpu_arm64.h#125
https://chromium.googlesource.com/breakpad/breakpad/+/88d8114fda3e4a7292654bd6ac0c34d6c88a8121/src/google_breakpad/common/minidump_cpu_arm64.h#74

The old one has `regs` first in this struct, the new one has it last. Presumably the latter matches the struct from Microsoft's SDK. I suppose ideally this code would handle both but realistically I have seen very little usage of fp register data from minidumps historically.